$NetBSD: patch-ae,v 1.2 2009/05/08 13:39:09 taca Exp $

Unused and no-op function which passing FILE as parameter not pointer
to FILE.  It cause compile problem on FILE is defined as opaque data
structure.

--- helpers/ntlm_auth/SMB/smbval/smblib-util.c.orig	2009-05-06 20:11:36.000000000 +0900
+++ helpers/ntlm_auth/SMB/smbval/smblib-util.c
@@ -61,6 +61,7 @@ static char *SMB_Prots[] =
     "NT LANMAN 1.0",
     NULL};
 
+#if 0
 /* Print out an SMB pkt in all its gory detail ... */
 
 void
@@ -79,6 +80,7 @@ SMB_Print_Pkt(FILE fd, RFCNB_Pkt * pkt, 
     /* etc                                                 */
 
 }
+#endif
 
 /* Convert a DOS Date_Time to a local host type date time for printing */
 
